neo4j/src/main/java/org/hibernate/ogm/datastore/neo4j/embedded/impl/EmbeddedNeo4jGraphDatabaseFactory.java

```diff
@@ -64,32 +66,33 @@ public GraphDatabaseService create() {
 		AtomicBoolean isNew = new AtomicBoolean( false );
 		FactoryHolder factoryHolder = GRAPH_DATABASE_SERVICE_MAP.computeIfAbsent( dbLocationAbsolutePath, ( String dbPath ) -> {
 			LOG.infof( " Create new service instance for dbPath  %s", dbLocationAbsolutePath );
-			GraphDatabaseService service = buildGraphDatabaseService();
+			GraphDatabaseFacade service = buildGraphDatabaseService();
 			FactoryHolder holder = new FactoryHolder();
 			holder.setCounter( 1 );
-			holder.setGraphDatabaseService( service );
+			holder.setGraphDatabaseFacade( service );
```

Comment:
When you send a PR don't rename a method and add changes in the same commit. It makes it harder to figure out where the important changes in this commit are

Reply:
Ok. I will try.